api/envoy/extensions/stat_sinks/open_telemetry/v3/open_telemetry.proto

```diff
@@ -44,4 +44,10 @@ message SinkConfig {
   // If set to true, metric names will be represented as the tag extracted name instead
   // of the full metric name. Default value is true.
   google.protobuf.BoolValue use_tag_extracted_name = 5;
+
+  // If set, emitted stats names will be prepended with a prefix, so full stat name will be
+  // <stats_prefix>.<stats_name>. For example, if the stat name is "foo.bar" and stats_prefix is
+  // 'pre', the full stat name will be "pre.foo.bar". If this field is not set, there is no
+  // prefix added. According to the example, the full stat name will remain "foo.bar".
+  string stats_prefix = 6;
```

Comment:
I think `stat_prefix` (singular) is the commonly used field name across the API surface, but that usually refers to something stylized like `tcp.<stat_prefix>.foo`. I think here maybe just go with `prefix` as per other stats syncs as it's a universal prefix.

Reply:
Also, the semantics of the other sinks seems to imply the `envoy` prefix is replaced, see https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto#envoy-v3-api-field-config-metrics-v3-statsdsink-prefix. Does this apply with OTLP stats?

Reply:
I renamed it. In the OTLP sink there's no default prefix.